server/depot.go

```diff
@@ -98,14 +112,93 @@ func (d fileDepot) Serial() (*big.Int, error) {
 		}
 		return s, nil
 	}
-	data, err := ioutil.ReadFile(name)
+	file, err := os.Open(name)
 	if err != nil {
 		return nil, err
 	}
-	serial := s.SetBytes(data)
+	defer file.Close()
+	r := bufio.NewReader(file)
+	// is there a better way??
+	data, err := r.ReadString('\r')
+	data = strings.TrimSuffix(data ,"\r")
```

Comment:
i was just wondering if ReadString('\n') or an equivalent function could not directly remove newlines as supposed but did not find a solution. i guess this is a slightly clumsy but working version we could live with